server/cli/kv.go

```diff
@@ -68,14 +71,17 @@ func runGet(cmd *commander.Command, args []string) {
 	key := proto.Key(args[0])
 	resp := &proto.GetResponse{}
 	if err := kv.Call(proto.Get, proto.GetArgs(key), resp); err != nil {
-		fmt.Fprintf(os.Stderr, "get failed: %s\n", err)
-		os.Exit(1)
+		fmt.Fprintf(osStderr, "get failed: %s\n", err)
+		osExit(1)
+		return
+	}
+	if resp.Value == nil {
+		fmt.Fprintf(osStderr, "%s not found\n", key)
+		osExit(1)
```

Comment:
shouldn't "not found" and other errors give a different exit code, like with `grep`?